js/utility.js

```diff
@@ -24,6 +27,37 @@ export function unhideMultipleElements(array) {
     }
 }
 
+export function showBigPopUp(textArray, item) {
+    let paragraphs = document.querySelectorAll('.bigPopUp > p');
+
+    paragraphs[0].textContent = textArray[0];
```

Comment:
const paragraphs = document.querySelectorAll('.bigPopUp > p')


```suggestion
   for (let i = 0; i < textArray.length; i++) {
         paragraphs[i] = textArray[i];
   }
```